javascript/sdk/README.md

```diff
@@ -0,0 +1,24 @@
+# Prelude SDK
+
+Use this utility if you want direct access to the Prelude API.
+
+## Install
+
+```bash
+npm install @prelude/sdk
+```
+
+## Quick start
+
+The following example registers an account and gets the manifest:
+
+```typescript
+import * as Prelude from "@prelude/sdk";
+const service = new Prelude.Service({ host: "https://detect.prelude.org" });
+
+const credentials = await service.iam.newAccount("test@example.com");
+
+service.setCredentials(credentials);
+
+const manifests = await service.build.listManifest();
```

Comment:
swap 'manifests' to 'manifest' everywhere (weird word in general with the plural state of it - blame my journalism background!)

Reply:
lol